interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php

```diff
@@ -0,0 +1,279 @@
+<?php
+/**
+ * interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php contains group participants details view .
+ *
+ * This is the therapy group's participants detail screen for the chosen group.
+ *
+ * Copyright (C) 2016 Shachar&Amiel <shachar058@gmail.com> <amielboim@gmail.com>
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Shachar Zilbershlag <shachar058@gmail.com>
+ * @author  Amiel Elboim <amielboim@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+?>
+<?php require 'header.php'; ?>
+<main id="group-details">
+    <div class="container-group">
+        <div class="row">
+            <div id="main-component" class="col-md-8 col-sm-12">
+                <div class="row">
+                    <div class="col-md-8 col-sm-12">
+                        <ul class="tabNav">
+                            <li><a href="<?php echo $GLOBALS['rootdir'] . '/therapy_groups/index.php?method=groupDetails&group_id=' . $groupId; ?>"><?php echo xlt('General data');?></a></li>
```

Comment:
not yet addressed
bunch of items were not addressed in the interface/therapy_groups/therapy_groups_views/groupDetailsParticipants.php file